bundles/org.openhab.binding.insteon/src/main/resources/OH-INF/thing/x10.xml

```diff
@@ -13,6 +13,7 @@
 
 		<label>X10 Device</label>
 		<description>An X10 device such as a switch, dimmer or sensor.</description>
+		<semantic-equipment-tag>ControlDevice</semantic-equipment-tag>
```

Comment:
Not sure as it can also be a sensor. Should we remove the tag in that case?

Reply:
Ok. Removing..

Reply:
@andrewfg If going that route, you would have to remove the tag for Insteon `device` and `legacy-device` thing type as well since both can be sensors as well.